queries/kconfig/highlights.scm

```diff
@@ -0,0 +1,79 @@
+"source" @include
+
+[
+  "mainmenu"
+  "config"
+  "menuconfig"
+  "choice"
+  "endchoice"
+  "comment"
+  "menu"
+  "endmenu"
+  "prompt"
+  "default"
+  "range"
+  "help"
+  (optional)
+  (modules)
+] @keyword
+
+[
+  "if"
+  "endif"
+  "depends on"
+  "select"
+  "imply"
+  "visible if"
+] @conditional
+
+[
+  "def_bool"
+  "def_tristate"
+] @keyword.function
+
+[
+  "||"
+  "&&"
+  "="
+  "!="
+  "<"
+  ">"
+  "<="
+  ">="
+  "!"
+] @operator
+
+[
+  "bool"
+  "tristate"
+  "int"
+  "hex"
+  "string"
+] @type.builtin
+
+[ "(" ")" ] @punctuation.bracket
+
+(macro_variable ["$(" ")"] @punctuation.special)
+
+(symbol) @variable
```

Comment:
symbol is kconfig's "notation" for anything used as an identifier pretty much

Reply:
This might help:

![image](https://github.com/nvim-treesitter/nvim-treesitter/assets/29718261/abbee435-27ac-4e6c-ae9b-57711dd4881f)
![image](https://github.com/nvim-treesitter/nvim-treesitter/assets/29718261/2b8fb92d-e44b-42d1-89f0-716ad0c2e154)